Makefile

```diff
@@ -283,6 +283,11 @@ destroy_all_nodes_from_namespaces:
 destroy_all_nodes:
 	skipper run $(SKIPPER_PARAMS) 'discovery-infra/delete_nodes.py --delete-all'
 
+deploy_ibip: _test_setup
+	skipper make $(SKIPPER_PARAMS) _deploy_nodes $(SKIPPER_PARAMS) ADDITIONAL_PARAMS="'--bootstrap-in-place'" NUM_WORKERS=0 NUM_MASTERS=1 NAMESPACE_INDEX=0
+
+wait_for_ibip_installation_completion:
```

Comment:
why not to make it as part of deploy ibip ?

Reply:
@RazRegev PR is adding platform field i think we just need to move to it and not to use new flag as --bootstrap-in-place.
What do u think? https://github.com/openshift/assisted-test-infra/pull/356/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R97

Reply:
It's not a platform, BIP is an installation mode
And I also think that we shouldn't make it similar to the regular flow, not until it's integrated in AI

Reply:
As for integrating ``wait_for_ibip_installation_completion`` as a part of ``deploy_ibip``, done